data-sources/citizens/generate-citizen-count-csv.ts

```diff
@@ -85,4 +89,42 @@ async function getTimestamp(id: number): Promise<number> {
   return await PassportContract.methods.timestampOf(id).call()
 }
 
+function getTotalExpiredPassports(weekEndDate: Date, maxPassportID: number): number {
+  console.info('getTotalExpiredPassports')
+
+  const weekEndDateString: string = weekEndDate.toISOString().substring(0, 10)
+
+  let totalExpiredPassports = 0
+  for (let passportID = 0; passportID < maxPassportID; passportID++) {
+    console.info(`weekEndDate: ${weekEndDateString}, passportID: ${passportID}`)
+
+    // Fetch voting power data from the citizen's data CSV
+    const citizenFilePath: string = `output/citizen-${passportID}.csv`
+    console.info('Fetching citizen data:', citizenFilePath)
+    const file: File = fs.readFileSync(citizenFilePath)
+    const csvData = file.toString()
+    // console.info('csvData:\n', csvData)
+    Papa.parse(csvData, {
+      header: true,
+      skipEmptyLines: true,
+      dynamicTyping: true,
+      complete: (result: any) => {
+        // console.info('result:', result)
+        result.data.forEach((row: any, i: number) => {
+          if (row.week_end == weekEndDateString) {
+            console.info(`row.week_end ${row.week_end}, row.voting_power: ${row.voting_power}`)
+            if (row.voting_power < 1.5) {
```

Comment:
This is fine, because we know it is, but we should actually upgrade the passports moving forward have the "expired" logic stored on chain.